addons/project/static/src/js/project_calendar.js

```diff
@@ -0,0 +1,15 @@
+odoo.define('project.ProjectCalendarController', function (require) {
+"use strict";
+const CalendarController = require('web.CalendarController');
+
+
+const ProjectCalendarController = CalendarController.include({
+    _renderButtonsParameters() {
+        let buttonsParams = this._super.apply(this, arguments);
```

Comment:
In ES6, `this._super.apply(this, arguments)` can be replaced with `this._super(...arguments)` :-)